src/org/thoughtcrime/securesms/ConversationItem.java

```diff
@@ -680,6 +680,10 @@ private void setReactions(@NonNull DcMsg current) {
         reactionsView.clear();
       } else {
         reactionsView.setReactions(reactions.getReactions());
+        reactionsView.setOnClickListener(v -> {
+          if (eventListener == null) return;
+          eventListener.onReactionClicked(current);
```

Comment:
the following style is used most often in this project ...

```suggestion
          if (eventListener != null) {
            eventListener.onReactionClicked(current);
          }
```

... and it is indeed easier to read in many cases as there are no different exit cases